vuepress/docs/next/docs/curate/bundle-component-details.md

```diff
@@ -0,0 +1,433 @@
+---
+sidebarDepth: 2
+---
+
+# Bundle Component Descriptors 
+
+
+Elements such as simple widgets, page templates, groups, and static resources are classified as platform components on Entando. This page describes each of these elements with an example.
+
+Currently, Entando supports the following component types. 
+
+![component-types.png](./img/component-types.png)
+
+Each component is defined by a descriptor YAML located in the bundle `platform` directory, inside the corresponding type as listed below. During the bundle build and pack phase, the descriptors are added to build the Docker images for the bundle. Here is an example structure of a bundle project:
```

Comment:
@jyunmitch Yep, pack creates the docker image(s) so anything that is in the bundle has to be there in the pack step. Technically, I think MFEs and MSs get built (using npm, mvn, etc.) but platform components only get packed since they get deployed as-is and require no other processing.